components/tools/OmeroWeb/omeroweb/webclient/views.py

```diff
@@ -176,10 +173,13 @@ def custom_index(request, conn=None, **kwargs):
 ##############################################################################
 # views
 
+# from omeroweb.webgateway import LoginView
 
-def login(request):
+
+class WebclientLoginView(webgateway_views.LoginView):
```

Comment:
Not sure how we we would do that @aleksandra-tarkowska. Checking the headers and enforcing the same origin policy is done by the browser.